src/main/java/com/example/springboot/dto/CreateCartItemRequestDto.java

```diff
@@ -0,0 +1,14 @@
+package com.example.springboot.dto;
+
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
+import lombok.Data;
+
+@Data
+public class CreateCartItemRequestDto {
+    @NotNull
```

Comment:
it's better to ident annotated fields
```suggestion
    @NotNull
    private Long bookId;
    
    @NotNull
    @Min(value = 1)
    private Long quantity;
```